views/components/related-content/macro.njk

```diff
@@ -1,19 +1,34 @@
 {% from "components/article-teaser/macro.njk" import articleTeaser %}
+{% from "components/promo-card/macro.njk" import promoCard with context %}
 
-{% macro relatedContent(articles, title, allLink) %}
+{% macro relatedContent(articles, title, allLink, usePromo = false) %}
     <section class="related-content">
         <header class="related-content__header">
             <h2 class="related-content__title t--underline">{{ title }}</h2>
         </header>
         <div class="related-content__content">
             <ul class="flex-grid flex-grid--3up">
-                {% for article in articles | take(3) %}
+                {% for article in articles %}
                     <li class="flex-grid__item related-content__item">
-                        {{ articleTeaser({
-                            'title': article.trailText or article.title,
-                            'linkUrl': article.linkUrl,
-                            'summary': article.summary
-                        }) }}
+                        {% if usePromo %}
+                            {{ promoCard({
+                                "title": article.title,
```

Comment:
should `title` not also be `article.trailText or article.title`. Any rather than the `usePromo` flag could we check if we have an image to determine which style to use?

Reply:
The new field doesn't have a `trailText` attribute (the component itself at the CMS end determines whether to use the linked entry's title or an overridden one).

I did think about checking for an image but then it'd be a style mismatch as the promo card looks like this:

![image](https://user-images.githubusercontent.com/394376/66196351-43e0dd00-e690-11e9-85e6-a922564efff5.png)

... but the regular version like this:

![image](https://user-images.githubusercontent.com/394376/66196373-4cd1ae80-e690-11e9-8ed7-894594c56111.png)

So I think all three need to look the same. Ultimately we'll probably want to switch the above/regular version to be a promo card I guess, if we start adding images to the homepage news grid.